### PR TITLE
add model info

### PR DESCRIPTION
**Pull request by Greg-Ma**
starting from release medusa-plus, discovery service publishes a field called product_model which can be directly used.

### CODE REVIEW

**Comment by Greg-Ma** [comment]
we verified that the package.json works for node 10.x 12.x and 14.x
`const bonjour = require("bonjour")({interface: allInterfaces()});`  
does not work on current version of bonjour any more, the interface is now a string instead of a string array.

**Comment by kseniadumpling** [comment]
There is a bunch of automated tests (`nmp run testcafe`, if I'm not mistaken), I recommend to run them before the merge :)

**Review by kseniadumpling — APPROVED** [review]

**Comment by kseniadumpling** [comment]
@Greg-Ma I shared write access with you. And removed myself from repo admin list. Good luck with BabyGroot :)
@vmt42 JFYI

**Pull request merged**
